fastcrawler_ui/routers/spiders.py

```diff
@@ -1,12 +1,30 @@
 from fastapi import APIRouter, Depends
 from fastcrawler import FastCrawler
 
+from fastcrawler_ui.controllers.spider import CrawlerController
 from fastcrawler_ui.core.fastapi.depends import get_crawler
+from fastcrawler_ui.repository.spiders import CrawlerRepository
 
 router = APIRouter()
+crawler_repository = CrawlerRepository()
```

Comment:
Don't use global variables in router. initiate it inside the controller if you have to.